manifest

```diff
@@ -11,13 +11,22 @@ export DOCUMENTATION_URL="https://chimeraos.org/about"
 export BUG_REPORT_URL="https://github.com/ChimeraOS/chimeraos/issues"
 
 export PACKAGES="\
-	gnome-desktop \
 	gnome-control-center \
 	gnome-session \
 	gnome-shell \
 	gnome-software \
 	gnome-terminal \
+	gnome-keyring \
+	gnome-menus \
+	gnome-system-monitor \
 	nautilus \
+	sushi \
```

Comment:
Thumbnails already appear to work, how is this different/better than the default?

Reply:
We could do without it. It is also the thumbnailer if I'm correct.

Reply:
I don't think so. Thumbnails work on the current build without sushi.

Reply:
Ok.